### ISSUE TITLE
How to obtain current GL context under X11?

### ISSUE THREAD

**Issue by sztomi** [sciter-sdk/rust-sciter#81]
I'm trying to port the [opengl behavior example](https://github.com/c-smile/sciter-sdk/blob/master/include/behaviors/behavior_drawing-opengl.cpp) to Rust under X11: [gist](https://gist.github.com/sztomi/b9f82e48823e7e334bda2254e93f5089)

This compiles, but `glXGetCurrentContext` always returns NULL, so none of the drawing actually runs. What am I doing wrong? Is there a sciter API to get the current context instead of the platform-specific one?

**Comment by pravic** [comment]
https://sciter.com/forums/topic/porting-opengl-behavior-to-linux-rust/

Isn't Rust-specific.

**Comment by sztomi** [comment]
Yeah, thanks. I opened the other topic as well. I'd close this, but I see you added labels. Do you want to keep it open?

**Comment by pravic** [comment]
No preferences. Feel free to decide)
